### PR TITLE
[RI1][RI2] Correctly link RI1 and RI2 from common index

### LINKED ISSUE #2873: [RI1][RI2] Correctly link RI1 and RI2 from common index
Now that both RI1 and RI2 are rst based, we can link both pieces together using proper rst directives.

### PR DESCRIPTION
**Pull request by gkunz**
Now that both RI1 and RI2 are rst based, we can link both pieces
together using proper rst directives. This also enables numbering
of RI1 headings.

Fixes #2873

Signed-off-by: Georg Kunz <georg.kunz@ericsson.com>

### CODE REVIEW

**Review by xudan16 — APPROVED** [review]
LGTM

File: doc/ref_impl/cntt-ri/index.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Reference Implementation 1
-==========================
+Reference Implementation 1 - OpenStack
+======================================
 
 This is the OPNFV based Reference Implementation (RI-1)
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri/index.rst:4]:
```suggestion
This is the OpenStack based Reference Implementation (RI-1)
```

**Comment by collivier** [comment]
@gkunz would you mind adding gates here too (see the common files in RC which are also verified by Jenkins)?

**Comment by gkunz** [comment]
@collivier do you mean a tox config for both RI1 and RI2?

**Review by CsatariGergely — APPROVED** [review]

**Comment by collivier** [comment]
@gkunz yes. it would help covering this files until the root document is ready. Maybe too much for RI but it makes sens for RC..

**Review by collivier — APPROVED** [review]

**Comment by gkunz** [comment]
> @gkunz yes. it would help covering this files until the root document is ready. Maybe too much for RI but it makes sens for RC..

Yes, I can add a few more build files. At some point, instead of having various copies of these files littered through the repo, we could think about have one set of master documents and then just create symlinks.

**Pull request merged**